.azure-pipelines/stage/publish.yml

```diff
@@ -158,6 +167,44 @@ jobs:
   - script: sudo .azure-pipelines/docker/save_cache.sh /mnt/docker_cache
     displayName: "Cache/save (publish_docker)"
 
+- job: docker_verify
```

Comment:
move this to existing verify job and add comment about improving docker publishing